crates/bevy_render/src/camera/camera.rs

```diff
@@ -219,37 +226,72 @@ impl Camera {
     /// The projection matrix computed using this camera's [`CameraProjection`].
     #[inline]
     pub fn projection_matrix(&self) -> Mat4 {
-        self.computed.projection_matrix
+        self.computed.projection_matrix.unwrap_or_default()
+    }
+
+    #[inline]
+    fn checked_projection_matrix(&self) -> Result<Mat4, BadProjectionMatrixError> {
```

Comment:
Wouldn't the other way around be more aligned with other Rust and Bevy functions?
I mean, that the "normal" function `projection_matrix` returns an `Option` or `Result` and a `projection_matrix_unchecked` returns the value without any checks?

That probably breaks some existing code, but should be fairly easy to change there, if it's documented in the upgrade guide.

Reply:
Fixed